docs/troubleshoot/upgrade-zowe-no-downtime.md

```diff
@@ -0,0 +1,91 @@
+# Upgrade Zowe in High availability
+
+As a systems programmer, you will need to upgrade Zowe regularly, to make sure you are using the latest features and to make sure you are using the latest security patches.
```

Comment:
... Zowe regularly. In case of maintenance version to get security updates and in active version also to have new features you are interested in.